test/integration.test.ts

```diff
@@ -169,4 +168,31 @@ describe.only('PNS Registry', () => {
     console.log("Emma's balance now:::", weiToEth(emmaBalance));
     console.log("Joe's balance now:::", weiToEth(joeBalance));
   });
+
+  it('Should add address to resolver', async () => {
+    const [joe, emma] = accounts.slice(1, 5);
+    //since joe has a record on the registry
+    const owner = await pnsResolverContract.connect(joe).getOwner(phoneNumber1);
+    assert.equal(owner, emma.address);
+
+    //get eth record
+    let getRecord = await pnsResolverContract['addr(bytes32)'](phoneNumber1);
+    assert.equal(getRecord, joe.address);
+
+    // change eth record to resolve emma
+    await pnsResolverContract.connect(emma)['setAddr(bytes32,address)'](phoneNumber1, emma.address);
+    getRecord = await pnsResolverContract['addr(bytes32)'](phoneNumber1);
+
+    assert.equal(getRecord, emma.address);
```

Comment:
Since you're calling the Resolver, shouldn't this variable name be getResolver instead of getRecord?